subprojects/xcfa/xcfa/src/main/java/hu/bme/mit/theta/xcfa/model/Builders.kt

```diff
@@ -81,6 +83,7 @@ constructor(
   private val edges: MutableSet<XcfaEdge> = LinkedHashSet(),
   val metaData: MutableMap<String, Any> = LinkedHashMap(),
   unsafeUnrollUsed: Boolean = false,
+  var prop: Expr<BoolType> = True(),
```

Comment:
Without understanding the actual needs of the ASG/Lasso checkers, it does not seem to be a good idea to me to attach a property to the model. Is not there a better solution?

Reply:
@AdamZsofi

Reply:
(memo: talked about it in meeting, not a priority, but opened an issue for later fixing: https://github.com/ftsrg/theta/issues/414 )